### ISSUE TITLE
bs-select can not be disabled

### ISSUE THREAD

**Issue by basz** [kaliber5/ember-bootstrap#76]
bs-select can not be disabled

```
{{bs-select prompt="Select one..." content=model.showTypeOptions disabled=true}}
```
Does not render a disabled attribute.

After adding this to your node_modules/ember-bootstrap/addon/components/bs-select.js it does disable.

```
attributeBindings: ['disabled']
```

I have the feeling I am missing something? Is this intentionally because Ember.Select does not support that? http://stackoverflow.com/questions/11457206/emberjs-how-to-disable-ember-select Workaround?

**Comment by simonihmig** [comment]
No, this was not intentional. The `disabled` property was added recently, bs-select needs a fix as suggested.

**Comment by simonihmig** [comment]
Forget the last comment, I missed the fact that the `disabled` property is correctly implemented in bs-select in the latest version 0.7.0. Make sure that you are using this version!

**Comment by basz** [comment]
upgrading fixes my probems, thanks!
